data.sql

```diff
@@ -1,25 +0,0 @@
-INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(
-	' Agumon',
-	'Feb 3, 2020',
-	0,
-	'1',
-	10.23
-),(
-	'Gabumon',
```

Comment:
- [ ] You have created the file with the correct code but removed them and submitted the empty one. Insert these codes again and resubmit the PR. 👍